.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+<!--
+Thank you for contributing to the Dataverse Project!
+
+NOTE: Make sure you search the issue tracker to verify this bug has not been submitted already.
+
+Issue tracker: https://github.com/IQSS/dataverse/issues
+
+If there is an existing open issue that matches the issue you want to report, please add a comment to it.
+
+If there is no pre-existing issue or it has been closed, please proceed with creating this bug issue.
+
+WARNING: If this is a security issue it should be reported privately to security@dataverse.org
+
+More information on bug issues and contributions can be found in the "Contributing to Dataverse" page:
```

Comment:
I think this is good - if people are interested in getting more detail they can go to the docs